### PR TITLE
include cardboard box for systems tests

### PR DESCRIPTION
**Pull request by stevedanomodolor**
This just includes the cardbox sdf that is used for system tests obstacle.

### CODE REVIEW

**Comment by SteveMacenski** [comment]
Can't you pull this from the fuel database? https://app.gazebosim.org/OpenRobotics/fuel/models/Cardboard%20box

```
<include>
<uri>
https://fuel.gazebosim.org/1.0/OpenRobotics/models/Cardboard box
</uri>
</include>
```

**Comment by SteveMacenski** [comment]
I meant more in its use in system tests, but I see you're spawning it so I think yeah this probably makes sense unless you hardcoded that SDF in the launch file itself

**Comment by stevedanomodolor** [comment]
> Can't you pull this from the fuel database? https://app.gazebosim.org/OpenRobotics/fuel/models/Cardboard%20box
> 
> ```
> <include>
> <uri>
> https://fuel.gazebosim.org/1.0/OpenRobotics/models/Cardboard box
> </uri>
> </include>
> ```

I get this error though, but does not crash anyways, it an issue with the model  itself. 
```

5: [ruby $(which gz) sim-1] [Err] [SystemPaths.cc:525] Could not resolve file [cardboard_box.png]

```

**Comment by SteveMacenski** [comment]
Actually @stevedanomodolor there's a models directory in the nav2 systems test package: https://github.com/ros-navigation/navigation2/tree/main/nav2_system_tests/models I think this should move over there instead

**Comment by stevedanomodolor** [comment]
> Actually @stevedanomodolor there's a models directory in the nav2 systems test package: https://github.com/ros-navigation/navigation2/tree/main/nav2_system_tests/models I think this should move over there instead
> 
> I think we can broadly delete the other things in that directory now with this repo

here I plan to do that, in the final cleanup stage, to remove all the modem and world there.

**Comment by SteveMacenski** [comment]
> here I plan to do that, in the final cleanup stage, to remove all the modem and world there.

I don't understand, can you rephrase? 

> how can I hardcode it during launch, can you give me an example, I still have to create the file separately right?

For example just having this trivial SDF as a string defined in a python variable and passing that into the spawner. But, I think having this SDF file makes sense, just should be in `nav2_system_tests/models` so its a testing asset

**Comment by stevedanomodolor** [comment]
> > here I plan to do that, in the final cleanup stage, to remove all the modem and world there.
> 
> I don't understand, can you rephrase?
> 
> > how can I hardcode it during launch, can you give me an example, I still have to create the file separately right?
> 
> For example just having this trivial SDF as a string defined in a python variable and passing that into the spawner. But, I think having this SDF file makes sense, just should be in `nav2_system_tests/models` so its a testing asset

Ok, Understood. Although, what if we put all assets in this repository and after porting to the new gazebo, delete the urdf and world folder in the nav2_system_tests package?, or do you prefer to keep the cardboard in the nav2_system_tests package?

**Comment by SteveMacenski** [comment]
> Although, what if we put all assets in this repository and after porting to the new gazebo, delete the urdf and world folder in the nav2_system_tests package?, 

I think you can delete all of those now, we won't use them again, that's what this repo is for. But this repo is also a stand alone sim for the TB3/4 - not just Nav2 system testing (also our bringup or just other people that want to use TB3/4 using standard ROS tooling). 

> or do you prefer to keep the cardboard in the nav2_system_tests package?

I'd prefer that, so that the `models/` just has this (maybe more later?). This is a Nav2-system-testing-specific asset, so I think for organization that's where it belongs

**Comment by stevedanomodolor** [comment]
OK, then closing this pr then.
